docs/source/guides/configuration.rst

```diff
@@ -59,20 +59,22 @@ Environment-specific parameters
 - ``mst_enable`` enables or disables multisignature transaction support in
   Iroha. We recommend setting this parameter to ``false`` at the moment until
   you really need it.
-- ``mst_expiration_time`` the time period after which a not fully signed MST
-  is considered expired.
-- ``max_rounds_delay`` optional parameter specifying the maximum delay between
-  two consensus rounds.
-  When Iroha is staying idle, it gradually increases the delay to reduce CPU,
-  network and logging load.
+- ``mst_expiration_time`` is the time period (in minutes) after which a not
```

Comment:
is the time period, in which not fully signed transaction is considered expired (in minutes)?